docs/source/commands/dev.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: The Rover dev command
+description: Query multiple GraphQL APIs from localhost
```

Comment:
Okay, this might be a bit wacky, but what do you think about something like this:
```suggestion
description: Query a locally federated GraphQL API
```

Reply:
looks good!